syntax = "proto3";

option go_package = "github.com/Lucino772/envelop/pkg/protobufs";

import "google/protobuf/empty.proto";
import "protobufs/events.proto";

service System {
    rpc StreamEvents (google.protobuf.Empty) returns (stream Event);
}

